build/android/bytecode/java/org/brave/bytecode/BraveClassVisitor.java

```diff
@@ -159,6 +166,10 @@ private boolean shouldMakePublicMethod(String className, String methodName) {
                 mMakePublicMethods.entrySet()) {
             String entryClassName = entry.getKey();
             ArrayList<String> methodNames = entry.getValue();
+            // Why className.contains(entryClassName)? This seems erroneous
+            if (methodNames.contains(methodName) && className.equals(entryClassName) != className.contains(entryClassName)) {
+                   System.out.println("Warning: class name " + className + " may not be written correctly!");
```

Comment:
Do you get this warning? If not, we just need to change `className.contains` to `className.equals` below.